README.md

```diff
@@ -0,0 +1,52 @@
+# Cluster Version Operator (CVO)
+
+## Building and Publishing CVO
+
+```sh
+./hack/build-image.sh && REPO=<your personal repo (quay.io/ahbinavdahiya | docker.io/abhinavdahiya)> ./hack/push-image.go
+```
+
+This builds image using `podman` locally using following command:
+
+```sh
+podman build -t "cluster-version-operator:${VERSION}" -f Dockerfile
```

Comment:
Do we care?  Folks who do can look in the script.  Folks who don't can ignore these details.  I'd rather drop this shell snippet and use:

> This builds image locally and then pushes `${VERSION}` and `latest` tags to `${REPO}/origin-cluster-version-operator`.

Along with some talk about `${VERSION}`; maybe:

> `${VERSION}` encodes the Git commit used to build the images.